homeassistant/components/zwave_js/strings.json

```
@@ -151,5 +151,193 @@
     }
+  },
+  "services": {
+    "clear_lock_usercode": {
+      "name": "Clear lock usercode",
+      "description": "Clears an usercode from a lock.",
+      "fields": {
+        "code_slot": {
+          "name": "Code slot",
+          "description": "Code slot to clear code from."
+        }
+      }
+    },
+    "set_lock_usercode": {
+      "name": "Set lock usercode",
+      "description": "Sets an usercode on a lock.",
+      "fields": {
+        "code_slot": {
+          "name": "[%key:component::zwave_js::services::clear_lock_usercode::fields::code_slot::name%]",
+          "description": "Code slot to set the code."
+        },
+        "usercode": {
+          "name": "Code",
+          "description": "Lock code to set."
+        }
+      }
+    },
+    "set_config_parameter": {
+      "name": "Set device configuration parameter",
+      "description": "Changes configuration parameters of your Z-Wave devices.",
+      "fields": {
+        "endpoint": {
+          "name": "Endpoint",
+          "description": "The configuration parameter's endpoint."
+        },
+        "parameter": {
+          "name": "Parameter",
+          "description": "The name (or ID) of the configuration parameter you want to configure."
+        },
+        "bitmask": {
+          "name": "Bitmask",
+          "description": "Target a specific bitmask (see the documentation for more information)."
+        },
+        "value": {
+          "name": "Value",
+          "description": "The new value to set for this configuration parameter."
+        }
+      }
+    },
+    "bulk_set_partial_config_parameters": {
+      "name": "Bulk set partial configuration parameters (advanced).",
+      "description": "Allows for bulk setting partial parameters. Useful when multiple partial parameters have to be set at the same time.",
+      "fields": {
+        "endpoint": {
+          "name": "[%key:component::zwave_js::services::set_config_parameter::fields::endpoint::name%]",
+          "description": "[%key:component::zwave_js::services::set_config_parameter::fields::endpoint::description%]"
+        },
+        "parameter": {
+          "name": "[%key:component::zwave_js::services::set_config_parameter::fields::parameter::name%]",
+          "description": "[%key:component::zwave_js::services::set_config_parameter::fields::parameter::description%]"
+        },
+        "value": {
+          "name": "[%key:component::zwave_js::services::set_config_parameter::fields::value::name%]",
+          "description": "The new value(s) to set for this configuration parameter. Can either be a raw integer value to represent the bulk change or a mapping where the key is the bitmask (either in hex or integer form) and the value is the new value you want to set for that partial parameter."
+        }
+      }
+    },
+    "refresh_value": {
+      "name": "Refresh value(s)",
```

```suggestion
      "name": "Refresh values",
```
Plural in brackets doesn't work in some languages. That's why I changed it to plural. If it is important to mention both, I think we should say refresh one or more values.